jenkins/job-configs/kubernetes-jenkins/kubernetes-verify.yaml

```diff
@@ -0,0 +1,85 @@
+- job-template:
+    name: 'kubernetes-verify-{suffix}'
+    description: |
+        Grab the latest from GitHub, then run unit and integration tests.<br>
+        Test Owner: Build Cop
+    properties:
+        - build-discarder:
+            num-to-keep: 200
+    node: unittest
+    builders:
+        - shell: 'JENKINS_BUILD_STARTED=true bash <(curl -fsS --retry 3 "https://raw.githubusercontent.com/kubernetes/kubernetes/master/hack/jenkins/upload-to-gcs.sh")'
+        - shell: |
+            export KUBE_FORCE_VERIFY_CHECKS='y'
+            export KUBE_VERIFY_GIT_BRANCH='{branch}'
+            export KUBE_TEST_SCRIPT="./hack/jenkins/verify-dockerized.sh"
+            timeout -k {kill-timeout}m {timeout}m ./hack/jenkins/gotest-dockerized.sh && rc=$? || rc=$?
+            {report-rc}
+    publishers:
+        - claim-build
+        - log-parser
+        - email-ext
+        - xunit:
```

Comment:
we probably want to remove the xunit publisher, since the verification checks don't (currently) produce junit.